bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/data/ModuleType.java

```diff
@@ -150,6 +158,13 @@
         }
     }
 
+    /**
+     * @return the UID compatible string of the enum name
+     */
+    private String toUID() {
+        return name().toLowerCase().replace("_", "-");
```

Comment:
Performance: Perhaps it would make sense to explicitly initialize and keep the UID strings in the enum (i.e. cached), so these string operations won't be called from many places, always processing the same conversions?

Reply:
Correct. Done.